scripts/startLocalFunctionsTestnet.js

```diff
@@ -0,0 +1,54 @@
+const process = require("process")
+const path = require("path")
+const fs = require("fs")
+const { startLocalFunctionsTestnet } = require("@chainlink/functions-toolkit")
+const { utils, Wallet } = require("ethers")
+// Loads environment variables from .env.enc file (if it exists)
+require("@chainlink/env-enc").config("../.env.enc")
+;(async () => {
+  const requestConfigPath = path.join(process.cwd(), "Functions-request-config.js") // @dev Update this to point to your desired request config file
```

Comment:
may be worth console logging 
console.log(`\nRequest config used is : ${ path.join(process.cwd(), "Functions-request-config.js")... }`

so devs can be left visual clues in case they get unexpected functions results